lib/livebook/application.ex

```diff
@@ -216,10 +216,18 @@ defmodule Livebook.Application do
   end
 
   # TODO: Remove in the future
-  defp update_app_secrets_origin do
-    for %{origin: :app} = secret <- Livebook.Secrets.get_secrets() do
-      {:ok, secret} = Livebook.Secrets.update_secret(secret, %{origin: {:hub, "personal-hub"}})
-      Livebook.Secrets.set_secret(secret)
+  defp migrate_secrets do
+    hub = Livebook.Hubs.fetch_hub!("personal-hub")
+
+    for %{name: name, value: value} <- Livebook.Storage.all(:secrets) do
```

Comment:
We should filter out secrets that do have `hub_id`, otherwise we always override them?

Reply:
We're migrating all old app secrets (or the updated to `{:hub, "personal-hub}` origin to the new struct, so it should be okay. WDYT?

Reply:
Ah we are saving them in a different storage namespace, in that case it would make sense to remove `:secrets` and that ensures we do it only once?